src/tokens/_shape.ts

```diff
@@ -0,0 +1,11 @@
+import type {TokenGroup} from './tokens';
+
+export const shape: TokenGroup = {
+  'border-radius-1': '0.2rem',
+  'border-radius-2': '0.4rem',
+  'border-radius-3': '0.8rem',
+  'border-radius-4': '1.2rem',
+  'border-radius-5': '1.6rem',
+  'border-radius-6': '2rem',
```

Comment:
These should use `px` values like our spacing values. If possible we should also follow the same number pattern used there for consistency.